shared_resource_creation_scripts/create_salmon_index.sh

```diff
@@ -34,11 +34,11 @@ else
 fi
 
 # Check that the reference genome for RNA was created successfully
-if [ -e GENOME_FASTA_GENERATION_COMPLETE ]
+if [ -e genome_reference ]
```

Comment:
@bryce-turner do you remember why you made this change?  Checking for the directory doesn't mean the genome generation is complete as the directory to completion can take hours I suspect, the flag is added as the last step of the genome script